ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -158,20 +187,18 @@ def aws_cleanup():
             "Wrong confirmation answer. Exiting"
         )
 
-    prefixes_hours_to_spare = defaults.CLUSTER_PREFIXES_TO_EXCLUDE_FROM_DELETION
+    prefixes_hours_to_spare = defaults.CLUSTER_PREFIXES_SPECIAL_RULES
+
+    if args.prefix:
+        for prefix, hours in args.prefix:
+            logger.info(
+                "Adding special rule for prefix '%s' with hours %s",
+                prefix, hours
+            )
+            prefixes_hours_to_spare.update({prefix: hours})
 
-    if not args.skip_prefixes:
-        prefixes_hours = input(
-            "Press Enter if there are no cluster prefixes to spare.\n"
-            "If you would like the cleanup to spare specific cluster prefixes, "
-            "please enter them along with the time allowed for these to be kept "
-            "running, in a dictionary representation.\nAn example: "
-            "{\'prefix1\': 36, \'prefix2\': 48}\" "
-        )
-        if prefixes_hours:
-            prefixes_hours_to_spare = eval(prefixes_hours)
     time_to_delete = args.hours[0][0]
-    assert time_to_delete > defaults.MINIMUM_CLUSTER_RUNNING_TIME_FOR_DELETION, (
+    assert time_to_delete > defaults.MINIMUM_CLUSTER_RUNNING_TIME, (
         "Number of hours is lower than the required minimum. Exiting"
     )
```

Comment:
I would also move the check for minimum number of hours to the `type` parameter of `parser.add_argument('--hours',...)`. It will be slightly more consistent and fails the command immediately and with better error. (Now it fails even after the confirmation question *Careful! This action could be highly destructive. Are you sure you want to proceed?* and it fails directly with full traceback - not only the clean error message.

Reply:
Done